Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -1525,11 +1539,20 @@ namespace PhysX
         const AZ::u8 subdivisionCount = m_shapeConfiguration.m_cylinder.m_subdivisionCount;
         const float height = m_shapeConfiguration.m_cylinder.m_height;
         const float radius = m_shapeConfiguration.m_cylinder.m_radius;
-        Utils::Geometry::PointList samplePoints = Utils::CreatePointsAtFrustumExtents(height, radius, radius, subdivisionCount).value();
 
-        const AZ::Vector3 scale = m_shapeConfiguration.m_cylinder.m_configuration.m_scale;
-        m_shapeConfiguration.m_cylinder.m_configuration
-            = Utils::CreatePxCookedMeshConfiguration(samplePoints, scale).value();
+        if (height > 0.0f && radius > 0.0f)
```

Comment:
do the same as above but split this into two separate error messages.

Reply:
Yep, updated to two error messages, thanks!